src/resources/pipe_posix.cc

```diff
@@ -451,6 +475,21 @@ PRIMITIVE(fork) {
   do {
     if (command == null) break;
 
+    for (int i = 0; environment && i < environment->length(); i += 2) {
+      Blob key;
+      environment->at(i)->byte_content(process->program(), &key, STRINGS_ONLY);
+      auto key_cstr = strndup(char_cast(key.address()), key.length());
```

Comment:
Do we not need to free the object?

Reply:
No, this is between fork and exec, so no leaks are possible.

Reply:
Please add a comment.